README.md

```diff
@@ -250,7 +250,8 @@ python -m venv venv && . venv/bin/activate
 python -m pip install --upgrade pip
 
 # install requirements
-pip install -r requirements.txt -r requirements-test.txt
+pip install django
```

Comment:
Considering that django is already specified in requirements, this is redundant.